### PR TITLE
TST: split chaining tests

### PR DESCRIPTION
**Pull request by jbrockmendel**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) for how to run them
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/tests/indexing/test_chaining_and_caching.py [hunk 1]
```diff
@@ -261,20 +289,32 @@ def random_text(nobs=100):
...
 
+    @pytest.mark.arm_slow
+    def test_detect_chained_assignment_is_copy(self):
+
         # an identical take, so no copy
         df = DataFrame({"a": [1]}).dropna()
         assert df._is_copy is None
         df["a"] += 1
 
+    @pytest.mark.arm_slow
```
Comment (jreback) [inline pandas/tests/indexing/test_chaining_and_caching.py:307]: might be worthile to move the slow to a separate file and just mark it

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
lgtm. small comment, ok for followon or here.

File: pandas/tests/indexing/test_chaining_and_caching.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/tests/indexing/test_chaining_and_caching.py:307]: im not wild about that pattern, but dont have a better alternative ATM.  im happy enough with how it is here

Comment (jreback) [reply pandas/tests/indexing/test_chaining_and_caching.py:307]: i c. maybe leave these decorators, but split anyhow just to make it more obvious?

Comment (jbrockmendel) [reply pandas/tests/indexing/test_chaining_and_caching.py:307]: i think the follow-up is to see if there's only a subset of these that are actually slow

**Pull request merged**